packages/vscode-extension/src/debug/prerequisitesHandler.ts

```diff
@@ -1186,6 +1186,16 @@ function checkNpmInstall(
         // treat unexpected error as installed
         error = err;
       }
+
+      const object = await fs.readJson(path.join(folder, "package-lock.json"));
+
+      ctx.properties[TelemetryProperty.BotbuilderVersion] =
+        object?.dependencies["botbuilder"]?.version;
+      ctx.properties[TelemetryProperty.TeamsFxVersion] =
+        object?.dependencies["@microsoft/teamsfx"]?.version;
+      ctx.properties[TelemetryProperty.TeamsJSVersion] =
+        object?.dependencies["@microsoft/teams-js"]?.version;
```

Comment:
Are these telemetries also targeted for v3? Please note that npm install cheking is removed in v3.

Reply:
Thanks for the comments! Adding check for V3, and the telemetry is currently only for v2. How to add V3 telemetry is under discussion.